socialhome/templates/base.html

```diff
@@ -50,9 +50,11 @@
                               <a class="nav-link" href="{% url 'account_logout' %}">{% trans "Logout" %}</a>
                           </li>
                       {% else %}
+                          {% if is_open_for_signup %}
```

Comment:
Please test things locally before submitting a PR ;)

If you check how the signup template/view figures out that the user cannot sign up, that should point you in the right direction. There is a clear message that comes up when going to `/accounts/signup/` when signups are closed, so the view knows that from somewhere.